.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+	"typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
```suggestion
	"typescript.tsdk": "node_modules/typescript/lib",
	"editor.formatOnSave": true,
	"editor.defaultFormatter": "oxc.oxc-vscode",
```

maybe add oxc as workspace recommendations

Reply:
What happens when someone doesn't have the VS Code extension installed?